### PR TITLE
Create hasTokenCounter attribute to assessment_config

### PR DESCRIPTION
**Pull request by jayjay19630**
### Create hasTokenCounter attribute to assessment_config
- [x] Create hasTokenCounter attribute to assessment_config
- [x] Render hasTokenCounter upon GET request

### CODE REVIEW

**Comment by RichDom2185** [comment]
Oh, I realised some tests are failing, could you fix them as well? We try to keep the coverage &ge; 95%. Thanks!

File: lib/cadet/courses/assessment_config.ex [hunk 1]
```diff
@@ -23,7 +24,7 @@ defmodule Cadet.Courses.AssessmentConfig do
 
   @required_fields ~w(course_id)a
   @optional_fields ~w(order type early_submission_xp
-    hours_before_early_xp_decay show_grading_summary is_manually_graded)a
+    hours_before_early_xp_decay show_grading_summary is_manually_graded has_token_counter)a
```
[the remark this answers is not in the document]
Comment (RichDom2185) [reply lib/cadet/courses/assessment_config.ex:27]: Nevermind, the existence of a default value means that it's fine for it to be optional (as long as that was the original intention).

Thanks!

**Comment by RichDom2185** [comment]
Corresponding frontend PR: https://github.com/source-academy/frontend/pull/2775

**Review by sayomaki — APPROVED** [review]
Looks good to me, keep up the good effort and contributions!

**Pull request merged**